implementations/README.md

```diff
@@ -1,11 +1,36 @@
 # Examples of implementations for metrics
 
 Notebooks, built on top of data collected by Perceval,
-for illustrating how to compute metrics:
+for illustrating how to compute different metrics of the Evolution Working Group. The notebooks will be explanatory in regard to the analysis performed. They should help the user understand how to calculate these metrics, as well as to understand how the metric is defined.
 
-* [Code Commits](Code_Commits.ipynb),
-includes the generation of file [git-commits.json],
-with a Perceval dump of a git repository, as a data source.
+The contents of this directory are the notebooks, as well as some scripts in the scripts directory. The scripts directory is a package containing the following modules:  
+
+- **Metric.py ([module](./code_df/metric.py) • [notebook](./notebooks_df/metric.ipynb))**:     
+This file contains the root class, Metric. All other classes inherit from it. It takes JSON data collected by Perceval and converts it into a form easier to analyze: like a dataframe, or a list of dictionaries.  
+
+- **Category modules**:  
+These classes provide basic functionality which is common to all metric classes working on the same category of data: commits, issues or pull request.
+The three Category modules are:
+    + **commit.py ([module](./code_df/commit.py) • [notebook](./notebooks_df/commit.ipynb))**
+    + **issue.py ([module](./code_df/issue.py) • [notebook](./notebooks_df/issue.ipynb))**
+    + **pullrequest.py ([module](./code_df/pullrequest.py) • [notebook](./notebooks_df/pullrequest.py))**
+
+- **Individual Metric modules**:  
+These are python scripts converted from the reference implementation notebooks. They inherit from the Category modules, but have the most important function of all --- to calculate values for the metrics they represent.   
+
+To summarize, the structure of the implementations is like so:
+```
+Root class (metric.py) <- Category classes (commit.py, for example) <- Metric classes
+```
+
+### Note
+- Every metric is calculated for the JSON file as a whole. If a JSON file has the data of only one repository, only that will considered for calculating the metric. Likewise, using a JSON file with the data of an entire project, 
```

Comment:
But metrics are not actually computed on JSON files, but on data structures with items... We read the JSON file when showing how to use the metric...

Reply:
OK, will make it clearer. Regarding the html file, I forgot to remove it. Will do so!